HealthSDK/GiniHealthSDKPinning/Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "GiniHealthSDKPinning",
+    platforms: [.iOS(.v12)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "GiniHealthSDKPinning",
+            targets: ["GiniHealthSDKPinning"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(name: "GiniHealthAPILibraryPinning", path: "../HealthAPILibrary/GiniHealthAPILibraryPinning"),
+        .package(name: "GiniHealthSDK", path: "../GiniHealthSDK"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages this package depends on.
+        .target(
+            name: "GiniHealthSDKPinning",
+            dependencies: [.product(name: "GiniHealthAPILibraryPinning", package: "GiniHealthAPILibraryPinning"),
+                           .product(name: "GiniHealthSDK", package: "GiniHealthSDK")]),
```

Comment:
I tested that it can be simplified to:
```suggestion
            dependencies: ["GiniHealthAPILibraryPinning", "GiniHealthSDK"]),
```